MAVProxy/mavproxy.py

```diff
@@ -1360,7 +1361,11 @@ def quit_handler(signum = None, frame = None):
 
     # open master link
     for mdev in opts.master:
-        if not mpstate.module('link').link_add(mdev, force_connected=opts.force_connected):
+        if mdev.find('?') != -1 or mdev.find('*') != -1:
```

Comment:
I don't get it - why not just always glob?

If those characters don't exist then the result will be the same.

Reply:
> I don't get it - why not just always glob?

it would break platforms like windows